### ISSUE TITLE
Sorting in the Admin UI only works on the item above or below.

### ISSUE THREAD

**Issue by leeonel** [keystonejs/keystone#4462]
### Expected behavior

Sorting in the Admin UI should work on any order.

### Actual/Current behavior

Sorting in the Admin UI only works on the item above or below

### Steps to reproduce the actual/current behavior

Enabling "sortable: true" in a `keystone.List()` and dragging the items.

### Environment

| Software         | Version
| ---------------- | -------
| Keystone         | 4.0.0-beta.5
| Node             | 6.11.3

**Comment by gluxon** [comment]
This may not be a keystone issue. I ran into this because I was adding new records in an update file asynchronously, causing the `sortOrder` field to not be unique. I switched to setting `sortOrder` manually to get this working, but synchronously adding new records would have the same effect.

tldr: check your collection items and make sure `sortOrder` is unique.
